sessions/sessions.go

```diff
@@ -111,16 +111,21 @@ func NewSMGeneric(cgrCfg *config.CGRConfig, rals, resS, thdS,
 }
 
 type SMGeneric struct {
-	cgrCfg             *config.CGRConfig             // Separate from smCfg since there can be multiple
-	rals               rpcclient.RpcClientConnection // RALs connections
-	resS               rpcclient.RpcClientConnection // ResourceS connections
-	thdS               rpcclient.RpcClientConnection // ThresholdS connections
-	statS              rpcclient.RpcClientConnection // StatS connections
-	splS               rpcclient.RpcClientConnection // SupplierS connections
-	attrS              rpcclient.RpcClientConnection // AttributeS connections
-	cdrsrv             rpcclient.RpcClientConnection // CDR server connections
-	smgReplConns       []*SMGReplicationConn         // list of connections where we will replicate our session data
-	Timezone           string
+	cgrCfg       *config.CGRConfig             // Separate from smCfg since there can be multiple
+	rals         rpcclient.RpcClientConnection // RALs connections
+	resS         rpcclient.RpcClientConnection // ResourceS connections
+	thdS         rpcclient.RpcClientConnection // ThresholdS connections
+	statS        rpcclient.RpcClientConnection // StatS connections
+	splS         rpcclient.RpcClientConnection // SupplierS connections
+	attrS        rpcclient.RpcClientConnection // AttributeS connections
+	cdrsrv       rpcclient.RpcClientConnection // CDR server connections
+	smgReplConns []*SMGReplicationConn         // list of connections where we will replicate our session data
+	Timezone     string
+	/* Part of new project
+	param               map[*rpc2.Client]struct{}
+	toBeRemovedSessions map[string][]*SMGSession
```

Comment:
toBeRemoved and RealActiveSession should be defined in function for sync, please remove from here